benchmark/cases/react_speed.py

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/ev python3
-# -*- coding: utf-8 -*-
-# ------------------------------------------------------------------------------
-#
-#   Copyright 2018-2019 Fetch.AI Limited
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-#
-# ------------------------------------------------------------------------------
-"""Example performance test using benchmark framework. test react speed on amount of incoming messages."""
-from benchmark.cases.helpers.dummy_handler import DummyHandler
-from benchmark.framework.aea_test_wrapper import AEATestWrapper
-from benchmark.framework.benchmark import BenchmarkControl
-from benchmark.framework.cli import TestCli
-
-
-def react_speed(benchmark: BenchmarkControl, amount: int = 1000) -> None:
```

Comment:
Why removed?

Reply:
the same test is made on a higher level and covers more part of the agent tested.

this was created earlier as for framework testing